### PR TITLE
Add example

### PR DESCRIPTION
**Pull request by OR13**
Screen Shot 2023-05-24 at 9 25 42 PM

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jwt/pull/89.html" title="Last updated on May 26, 2023, 4:03 PM UTC (15a08c4)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jwt/89/0b73db0...15a08c4.html" title="Last updated on May 26, 2023, 4:03 PM UTC (15a08c4)">Diff</a>

### CODE REVIEW

**Review by mprorock — APPROVED** [review]
awesome!

**Review by dlongley — APPROVED** [review]
This looks great!

File: index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
       src="https://www.w3.org/Tools/respec/respec-w3c"
       class="remove"
     ></script>
+    <script class="remove" src="https://cdn.jsdelivr.net/gh/transmute-industries/respec-vc-jwt@0.0.7/dist/main.js"></script>
```
Comment (andresuribe87) [inline index.html:10]: Should this be hosted by w3c?

**Review by andresuribe87 — COMMENTED** [review]
This is amazing!

File: index.html (hunk 1 above)
Comment (iherman) [reply index.html:10]: Yes, thanks for spotting this, @andresuribe87. The external script references to the specification text should be in the same directory (or subdirectory) relative to the spec text itself, and refer to it via relative URL.

Comment (OR13) [reply index.html:10]: I'm happy to fix this after it is fixed in the vc data model:

https://github.com/w3c/vc-data-model/blob/main/index.html#L14

**Pull request merged**